### PR TITLE
SQL: ODBC: document newest connection string parameters

### LINKED ISSUE #146: Installation prerequisites check
https://www.elastic.co/guide/en/elasticsearch/reference/current/sql-odbc-installation.html
The doc says: "If you fail to meet any of the prerequisites the installer will show an error message and abort the installation."

I installed ODBC driver 6.7.0 (esodbc-6.7.0-windows-x86) on a Windows Server 2012 R2 Datacenter machine (Windows Server 6.3 Build 9600) and it installs successfully without an error even though it does not meet the prerequisites.

### PR DESCRIPTION
**Pull request by bpintea**
This PR adds the documentation for two newly added connection string
parameters: `AutoEscapePVA` and `IndexIncludeFrozen`.

It also removes the recommended OSes from the prerequisites list and
places the recommendation distinctively: the unmet prerequisites will
fail the installation (as documented), while the driver would install on 
other OSes than those recommended.
Fixes https://github.com/elastic/elasticsearch-sql-odbc/issues/146

### CODE REVIEW

File: docs/reference/sql/endpoints/odbc/configuration.asciidoc [hunk 1]
```diff
@@ -354,3 +354,16 @@ it as the value for the column. If not set, the server will return an error.
...
+`AutoEscapePVA` (default: `true`)::
+The pattern-value arguments make use of `_` and `%` as special characters to
+build patern matching values. Some applications however use these chars as
+regular ones, which can lead to {es-sql} returning more data than the app
+intended. With the auto escaping, the driver will inspect the arguments and
+will escape these special characters if not already done by the application.
+
+
+`IndexIncludeFrozen` (default: `false`)::
+If this parameter is `true`, the server will allow quering of frozen indices.
```
Comment (astefan) [inline docs/reference/sql/endpoints/odbc/configuration.asciidoc]: Typo: I believe the word is `querying`.

Comment (astefan) [reply docs/reference/sql/endpoints/odbc/configuration.asciidoc]: Also, I would reformulate a bit to emphasize on the idea of frozen indices being "included" in the search (in addition to the usual normal indices).

```
If this paramter is `true`, the server will include frozen indices in the query execution.
```

**Review by astefan — APPROVED** [review]
LGTM. Left two comments.

File: docs/reference/sql/endpoints/odbc/configuration.asciidoc (hunk 1 above)
Comment (bpintea) [reply docs/reference/sql/endpoints/odbc/configuration.asciidoc]: Thanks, rephrasing applied.

**Pull request merged**